src/components/avatar-volume-controls.js

```diff
@@ -50,14 +50,17 @@ AFRAME.registerComponent("avatar-volume-controls", {
         return;
       }
 
-      const { audioOutputMode, globalVoiceVolume } = window.APP.store.state.preferences;
+      const { audioOutputMode, globalVoiceVolume, globalRolloffFactor } = window.APP.store.state.preferences;
       const volumeModifier = (globalVoiceVolume !== undefined ? globalVoiceVolume : 100) / 100;
       let gain = volumeModifier * this.data.volume;
       if (audioOutputMode === "audio") {
         this.avatarAudioSource.el.object3D.getWorldPosition(positionA);
         this.el.sceneEl.camera.getWorldPosition(positionB);
         const squaredDistance = positionA.distanceToSquared(positionB);
-        gain = gain * Math.min(1, 10 / Math.max(1, squaredDistance));
+        const distanceBasedAttenuation = Math.min(1, 10 / Math.max(1, squaredDistance));
+        // Clamp globalRolloffFactor to [0.0, 1.0]
+        gain = gain * (1.0 + Math.max(0.0, Math.min(1.0, globalRolloffFactor)) *
```

Comment:
Need to guard against `globalRolloffFactor` being `undefined`. If it is, `Math.min(1.0, undefined)` is `NaN`